python-avd/pyavd/_eos_designs/shared_utils/filtered_tenants.py

```diff
@@ -433,3 +432,39 @@ def get_additional_svi_config(svi_config: dict, svi: dict, vrf: dict) -> None:
                     ospf_keys.append({"id": ospf_key["id"], "hash_algorithm": ospf_key.get("hash_algorithm", "sha512"), "key": ospf_key["key"]})
                 if ospf_keys:
                     svi_config.update({"ospf_authentication": ospf_authentication, "ospf_message_digest_keys": ospf_keys})
+
+    @cached_property
+    def bgp_in_network_services(self: SharedUtils) -> bool:
+        """
+        True if VRF default is enabled for BGP under network services.
```

Comment:
I am not sure why we mention VRF default here?

Reply:
fixed